internal/cli/log_streams.go

```diff
@@ -296,7 +297,7 @@ func (c *cli) logStreamPickerOptionsByType(desiredType logStreamType) pickerOpti
 		}
 		if len(options) == 0 {
 			return nil, fmt.Errorf(
-				"There are currently no log streams of type: %q, use 'auth0 logs streams create %s' to create one.",
+				"there are currently no log streams of type: %q, use 'auth0 logs streams create %s' to create one",
```

Comment:
Should we use backticks around the command in these errors, like we're doing in other places?